lib/exercism/user_exercise.rb

```diff
@@ -51,6 +51,18 @@ def unarchive!
     update_attributes(archived: false)
   end
 
+  def request_help?
+    request_help
```

Comment:
What about renaming the property to `help_requested`?

Makes it more obvious to me that 'help' is the important concept.
(Particularly bearing in mind that 'request' is a common concept in a network environment.)

(but I think request_help! and remove_help! methods are still ok, particularly since you'll be able to see they operate on the 'help_requested' property)